pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -22,6 +22,21 @@ const (
 	auditPolicyAsset = "pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml"
 )
 
+// DefaultPolicy brings back the default.yaml audit policy to init the api
+func DefaultPolicy() ([]byte, error) {
+	cm, err := GetAuditPolicies("kube-apiserver-audit-policies", "openshift-kube-apiserver")
```

Comment:
`cluster-kube-apiserver-operator` could use `GetAuditPolicies` directly :)

Reply:
yeah, I agree, if we want to keep it in library-go we will need to make it reusable:
```
func DefaultPolicy(name, namespace string) ([]byte, error)
```